.travis.yml

```diff
@@ -15,8 +15,8 @@ env:
   - JOB=test
 
 script:
-  - 'if [ "$JOB" = "test" ]; then make test-ci; fi'
-  - 'if [ "$JOB" = "test-coverage" ]; then make test-ci-coverage; fi'
+  - 'if [ "$JOB" = "test" ]; then CI=true make test-ci; fi'
```

Comment:
You don't need to pass `CI` as a variable because it is set by Travis by default: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables.